Vic2ToHoI4/Data_Files/blankMod/output/common/ideas/_cultures.txt

```diff
@@ -36,10 +48,10 @@ ideas = {
 			picture = culture_afro_american
 			modifier = { }
 		}
-		culture_afro_antillean = {
+		culture_afro_antillean  = {
```

Comment:
Similarly these spaces. They aren't a big deal, but would be nice to remove. Again, you can use Notepad++'s find/replace:
1. Set the Search Mode to extended
2. Make the Find be ```(space)(space)=```
3. Set the Replace be ```(space)=```